### PR TITLE
[[ ModuleAssemblies ]] Support multi-module assemblies in standalones

### PR DESCRIPTION
**Pull request by livecodeali**
Requires https://github.com/livecode/livecode-ide/pull/1882

### CODE REVIEW

**Comment by livecodeali** [comment]
Test branch: https://github.com/livecode/livecode/pull/6273

File: tests/ide-support/standalonebuilder/assembly.livecodescript [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+private function _TestScriptForInclusion
+   set the itemdelimiter to slash
+   local tExternalTest
+   put the filename of me into tExternalTest
+   put "/_assembly.livecodescript" into item -1 of tExternalTest
+   return textDecode(url("binfile:" & tExternalTest), "utf-8")
+end _TestScriptForInclusion
+
+on TestStandaloneInclusions
+breakpoint
```
Comment (montegoulding) [inline tests/ide-support/standalonebuilder/assembly.livecodescript]: @livecodeali this probably shouldn't be here

File: tests/ide-support/standalonebuilder/assembly.livecodescript [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+   local tExtra
+   put "assemblytest-support.lcb" into tExtra["support_files"]
+   breakpoint
+   revIDEExtensionLoad "lcb", sExtensionName, \
+         __GetExtFolder(), "0.0.0", "installed", \
+         "", "true", "assemblytest.lcb", tExtra
+end __InstallExtension
+
+on TestTeardown
+   --StandaloneBuilderCleanUpStandalones
```
Comment (montegoulding) [inline tests/ide-support/standalonebuilder/assembly.livecodescript]: @livecodeali should this be uncommented

File: tests/ide-support/standalonebuilder/assembly.livecodescript (hunk 1 above)
Comment (livecodeali) [reply tests/ide-support/standalonebuilder/assembly.livecodescript]: gah

File: tests/ide-support/standalonebuilder/assembly.livecodescript (hunk 2 above)
Comment (livecodeali) [reply tests/ide-support/standalonebuilder/assembly.livecodescript]: I wondered about that - they all seem to be commented out (this is copy-pasted from another one)

**Comment by montegoulding** [comment]
@livecode-vulcan review ok b1afe0e

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok b1afe0e6b574b849b84748908740f05a16ac75a9

**Comment by livecode-vulcan** [comment]
😎 test success b1afe0e6b574b849b84748908740f05a16ac75a9

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1154)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1170)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1144)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/97)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/97)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1149)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1171)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1154)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1170)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1153)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1163)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1187)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1148)

**Pull request merged**